modules/cluster-logging-rn-5.5.6.adoc

```diff
@@ -0,0 +1,49 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-6_{context}"]
+= Logging 5.5.6
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0386[OpenShift Logging Bug Fix Release 5.5.6].
+
+[id="openshift-logging-5-5-6-bug-fixes"]
+== Bug fixes
+* Before this update, the Pod Security admission controller added the label `podSecurityLabelSync = true` to the `openshift-logging` namespace. This resulted in our specified security labels being overwritten, and as a result Collector pods would not start. With this update, the label `podSecurityLabelSync = false` preserves security labels. Collector pods deploy as expected. (link:https://issues.redhat.com/browse/LOG-3340[LOG-3340])
+
+* Before this update the Operator would install the console view plugin even when not enabled on the cluster, causing the Operator to crash. With this update, accounts for clusters that do not have the console enabled no longer crash the Operator. (link:https://issues.redhat.com/browse/LOG-3407[LOG-3407])
+
+* Before this update, a prior fix to support a regression where the status of the Elasticsearch deployment was not being updated caused the Operator to crash unless the `Red Hat Elasticsearch Operator` was deployed. With this update, that fix has been reverted so the Operator is now stable but re-introduces the previous issue related to the reported status. (link:https://issues.redhat.com/browse/LOG-3428[LOG-3428])
+
+* Before this update, when the Loki Operator deployed with a size of `1x.small` only one pod deployed for the LokiStack gateway. With this update, two pods deploy for the gateway. (link:https://issues.redhat.com/browse/LOG-3478[LOG-3478])
+
+* Before this update, records written to Elasticsearch would fail if multiple label keys had the same prefix and some keys included included dots. With this update, underscores replace dots in label keys, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3341[LOG-3341])
```

Comment:
```suggestion
* Before this update, records written to Elasticsearch would fail if multiple label keys had the same prefix and some keys included dots. With this update, underscores replace dots in label keys, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3341[LOG-3341])
```